debian/mumbledj.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+After=mumble-server.service
```

Comment:
It's just an ordering hint; if there is no `mumble-server.service` unit then it has no effect.